src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -112,7 +112,7 @@ public static function register($mode = 0)
             $trace = debug_backtrace(true);
             $group = 'other';
 
-            $isWeak = DeprecationErrorHandler::MODE_WEAK === $mode || (DeprecationErrorHandler::MODE_WEAK_VENDORS === $mode && $isVendor = $inVendors($file));
+            $isWeak = DeprecationErrorHandler::MODE_WEAK === $mode || (self::MODE_WEAK_VENDORS === $mode && $isVendor = $inVendors($file));
```

Comment:
Argh. Please reconsider… these few lines can save literal hours to anyone try to contribute to this error handler. I know the audience is small, but it exists.